src/views/dependencyDataProvider.ts

```diff
@@ -124,6 +124,14 @@ export class DependencyDataProvider implements TreeDataProvider<ExplorerNode> {
         const children = (!this._rootItems || !element) ?
             await this.getRootNodes() : await element.getChildren();
 
+        if (children && element instanceof ContainerNode) {
+            if (element.isMavenType()) {
```

Comment:
I think it makes sense to always sort alphabetically. Then there is no need check the type of the container. And we can remove `isMavenType()` & `isGradleType()`

Reply:
Done.

Reply:
@jdneo , how would you like me to name metaData for gradle, do we want to name them same as the maven one? maven.groupId, maven.artifactId, maven.version?

Reply:
Ah that's a good question.

I think still calling it `maven.xxx` does not make sense anymore. Since they are not persisted by maven importer. Meanwhile, a dependency may not come from maven registry.

what about just calling it `groupId `, `artifactId ` and `version`?

BTW, since it's specific with gradle importer, we can create an issue and discuss those implementation details there.